lib/run/index.js

```diff
@@ -158,16 +158,19 @@ module.exports = function (options, callback) {
                         summary: emitter.summary
                     });
 
-                    _.forEach(['environment', 'globals', 'collection'], function (item) {
+                    _.forEach(['environment', 'globals'], function (item) {
                         // fetch the path name from options if one is provided
-                        var path = _.get(options, _.camelCase(`export-${item}`));
+                        var path = _.get(options, _.camelCase(`export-${item}`)),
+                            exportContent = emitter.summary[item].toJSON();
 
                         // if the options have an export path, then add the item to export queue
                         path && emitter.exports.push({
                             name: item,
                             default: `newman-${item}.json`,
                             path: path,
-                            content: _.merge(emitter.summary[item].toJSON(), {
+                            content: _.merge(exportContent, {
+                                timestamp: Date.now(),
```

Comment:
Why do we have timestamp here now? I think it's irrelevant with respect to variable list.